### ISSUE TITLE
cell frequency?

### ISSUE THREAD

**Issue by HarmSluiman** [HarmSluiman/NNN#8]
It has been stated at times that we see at a frequency slightly below 50 hertz (5 cycles per second). This is the proposed explanation of why we don't see the flickering of a 60 hz electric light bulb, while some people see a 50 hz bulb and most see the flicker of a 40 hz or less.

For this project the cycle time could be a basis for a sampling rate of images and conveniently allow for some less random correlation when simulating in on a computer since it has a limited amount of parallel activity possible. This would otherwise require massive numbers of parallel threads/cores to properly simulate. perhaps a target for a super nervous system, but not a target for this project.

So what is the basis for this perceived frequency/cycle? And what is the frequency if it exists?

Nerves fire based on potential and physical chemicals and compounds are involved and to some degree consumed. Is the regenerative cycle needs to replace these consumed resources part of the cycle? A cell can't transmit when it's supply is diminished and it can hold only so much.

Given a cell holds the potential for a while as it triggers a transmission over the synapse, perhaps that actual firing of a synapse also triggers a regeneration or recovery cycle. Nutrients are supply by diffusion through the outer pigment layer and this may be a starting point for investigation.

For us to properly utilize and simulate this cycle, we need to know more about it's existence and behavior.

**Comment by Jsluiman** [comment]
(To be edited)
Neurons have a resting potential between -40 and -90 mV

Potential depends on sodium/potassium balance in/out of the cell.  
?Cells from the older neurons pass along the sodium/potassium ions to newer cells.

Neurons receive nutrition through aerobic metabolism and/then cellular respiration- turning oxygen into ATP (Adenosine triphosphate).  

?Glucose carries other elements like sodium to cells, this is how the brain is fed despite the blood barrier

**Comment by HarmSluiman** [comment]
Agreed that these are the mechanical factors that likely impact a cells ability to react to stimulation, and this is very likely an impact to the capabilities to continuously stimulated cells.
On further thought, while this likely impacts sensing cells such as cones and rods of the retina given they are continuously stimulated, the simple fact that potential for triggering will rise and fall as the stimulation travels across multiple cells/nerves, plus the physical distance itself, all add to any recognizable refresh time.

As a hypothetical comparison, a typical machine learning neural network will transit across 4-6 layers, which is often used to compare to the nervous system. On average, the node/nerve of each progressive layer is stimulated less frequently than the previous layer.

Based on this thinking, the cycle time may have an impact, and be the base of optimal throughput, but the more complex the "observation" the longer it will take. Any variation across people may well be a representation of different implementations of a particular pattern recognition.

If nothing else, the notes taken here show how fundamental basic health and nutrition impact the most fundamental parts of the human body every moment, not just over time. Consider a compromised nutrition system during a learning experience, and then how often even variation occurs in a person versus another person. It seems no wonder that everyone ends up different.
